packages/core/src/lib/runtime.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  WebpackRemoteScriptFactory,
+  WebpackSharingScopeFactory,
+} from '../integrations/webpack/factory';
+import type {
+  ModuleFederationRuntime,
+  ModuleFederationRuntimeOptions,
+} from '../types';
+
+export const createModuleFederationRuntime = (
```

Comment:
Where and how can we use this?

I think this will be great place for us to extend the default adding of remotes to Global scopes.

Reply:
Originally I was thinking someone could create it and throw it on global/window, and if its there we use it and if not create it. Still not clear to me how this may work, it feels like there are 2 runtime needs. One for embedding in the remoteEntry file, and one that replaces the global scope pollution - possibly also surfacing an API off of. 

If we think we want to version the runtime, could be nice to have the API under the runtime. This needs refinement but I left it was a conversation piece we can remove/change later